Makefile

```diff
@@ -237,6 +238,7 @@ cleanup-e2e-robot:
 ## e2e-robot: Run e2e robot tests
 e2e-robot:
 	pabot --verbose --processes ${ROBOT_THREADS} \
+		  -v NO_CLEAN_UP:${NO_CLEAN_UP} \
```

Comment:
indent

Reply:
On my PC, it's showing as should be:
![image](https://user-images.githubusercontent.com/49116723/116468263-507fe400-a879-11eb-99a4-e97243e0f964.png)